frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -409,7 +409,7 @@ const EnvFromPairElement = DragSource(DRAGGABLE_TYPE.ENV_FROM_ROW, pairSource, c
           <div className="col-xs-5 pairs-list__value-pair-field">
             <ValueFromPair pair={pair[EnvFromPair.Resource]} configMaps={configMaps} secrets={secrets} onChange={this._onChangeResource} disabled={readOnly} />
           </div>
-          <div className="col-xs-5 pairs-list__name-field">
+          <div data-test-id="env-prefix" className="col-xs-5 pairs-list__name-field">
```

Comment:
Thanks, this looks cleaner.

I'd move this on the `input` directly since that's what you need in the test.